src/Database/Relations/AttachMany.php

```diff
@@ -14,12 +14,16 @@ class AttachMany extends MorphManyBase
      * Create a new has many relationship instance.
```

Comment:
@RomainMazB Would you mind doing us a favour and adding the params to the docblock?

Reply:
Yep no problem. It should have be done without the need to ask :grimacing:

Reply:
These need type data and ideally an explanation of their purpose @RomainMazB

Reply:
The fact no docblock was implemented anywhere disappointed me about how I was supposed to do.

Reply:
@RomainMazB ?

Reply:
I've added an explicit comment on both `AttachMany` and `AttachOne` file.